File: 1619. Mean of Array After Removing Some Elements/solution.cpp
class Solution {
public:
  double trimMean(vector<int>& arr) {
    int fivePercent = arr.size() / 20;
    vector<int> adjustedArr = arr;
    sort(adjustedArr.begin(), adjustedArr.end());
    int totalSum = 0;
    for (int index = fivePercent; index < adjustedArr.size() - fivePercent; index++){
      int num = adjustedArr[index];
      totalSum += num;
    }

    return (double) totalSum / (adjustedArr.size() - (2 * fivePercent));
  }
};

File: 1619. Mean of Array After Removing Some Elements/solution.java
class Solution {
  public double trimMean(int[] arr) {
    int fivePercent = arr.length / 20;
    int[] adjustedArr = arr.clone();
    Arrays.sort(adjustedArr);
    int totalSum = 0;
    for (int index = fivePercent; index < adjustedArr.length - fivePercent; index++){
      int num = adjustedArr[index];
      totalSum += num;
    }

    return (double) totalSum / (adjustedArr.length - (2 * fivePercent));
  }
}